go/runtime/transaction/batch.go

```diff
@@ -2,8 +2,3 @@ package transaction
 
 // RawBatch is a list of opaque bytes.
 type RawBatch [][]byte
-
-// String returns a string representation of a batch.
-func (b RawBatch) String() string {
```

Comment:
This could result in a lot of verbosity in logs.

Reply:
It's not that bad, the transactions are displayed base64 encoded. The amount of logs is comparable to the amount of `received peer message` logs (where we log the message contents as well). Helps a lot with debugging though.

Reply:
Ok.